README.md

```diff
@@ -2,6 +2,45 @@
 ![Coded by humans](https://img.shields.io/badge/Coded%20by%20humans-100%25-blue)
 [![GH Pages Build](https://github.com/nicolaerario/fake-it-till-you-make-git/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/nicolaerario/fake-it-till-you-make-git/actions/workflows/pages/pages-build-deployment)
 
-# Fake it till you make `Git`
+# _Fake it till you make_ `Git`
 
-An (opinionated) approach to `Git` and `GitHub`
+Welcome to "Fake it till you make `Git`" - a resource dedicated to educating people on how to use `Git` and `Github`. This repository is published via `Github Pages` and can be found on [this link](https://nicolaerario.github.io/fake-it-till-you-make-git/), and we hope to help `you` become proficient in using `Git` and `Github`.
+
+## _How to contribute_
+
+We welcome `contributions` to this resource from anyone who is interested in helping us `improve` and `expand` our content. If you're interested in contributing, please take a look at our Github repository [issues page](https://github.com/nicolaerario/fake-it-till-you-make-git/issues) and create a `new issue` or make a `pull request` for an existing one. We accept contributions in the form of `bug reports`, `website improvements`, and `new content`. We also welcome `translations` of our existing content into other languages.
+
+## _Create and Edit a_ `Resource Article`
```

Comment:
I think I will write the paragraph of the guide myself: maybe is better to point users to request new features with issues.
Or they could write a paragraph spontaneously and it will be considered for publication.